pkgs/mime/tool/media_types.dart

```diff
@@ -37,8 +37,8 @@ void main() {
   final buf = StringBuffer(prefix);
   buf.writeln();
   buf.write('''
-| MIME type | Default | Additional |
-| --- | --- | --- |
+| MIME type                                | Default     | Additional          |
+| ---                                      | ---         | ---                 |
```

Comment:
Just use dashes all the way here.
Three dashes are not more significant than one, so it looks random to have ` ---      `.
I'd drop the lading/trailing space as well, and just fill the space up with `-`s.

```markdown
|----------------------------------------|-----------|-------------------|
```

(The only characters which are significant is at least one `-`, and a leading and/or trailing `:` to guide alignment. We could use left alignment on all columns, to match the source layout, by changing the first `-` to a `:`.)